scripts/provisioning-tests

```diff
@@ -111,11 +111,11 @@ while ! kubectl --kubeconfig /etc/rancher/k3s/k3s.yaml rollout status -w -n catt
 done
 #kill $TPID
 
-echo Running provisioning-tests
-export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
-go test -v -timeout 60m ./tests/integration/pkg/tests/... || {
-    echo -e "-----RANCHER-LOG-DUMP-START-----"
-    cat /tmp/rancher.log | gzip | base64 -w 0
-    echo -e "\n-----RANCHER-LOG-DUMP-END-----"
-    exit 1
-}
+# echo Running provisioning-tests
```

Comment:
Why not just remove this bash code instead of commenting it out? Will we need it for k8s 1.25? If not remove, at least add a comment explaining why it's commented out and when to reverse this

Reply:
This is only temporary per https://github.com/rancher/rancher/pull/38113#issuecomment-1169506170